cmd/akamai-purger/main.go

```diff
@@ -150,19 +169,21 @@ func main() {
 	cmd.FailOnError(err, "Unable to setup Akamai purger gRPC server")
 	akamaipb.RegisterAkamaiPurgerServer(grpcSrv, &ap)
 
-	go cmd.CatchSignals(logger, grpcSrv.GracefulStop)
+	go cmd.CatchSignals(logger, func() {
+		grpcSrv.GracefulStop()
+		// Stop the ticker and signal that we want to shutdown by writing to the
```

Comment:
This comment isn't completely true because we now do a blocking write on stop, the timeout will be 15s + if there is currently a `ap.purge()` running however long that takes to return (which isn't actually bounded).

Reply:
(I believe this is why I originally didn't use an unbuffered channel)

Reply:
I think this is now true again since I restored buffering to the chans.